test/snapshots/seam-blueprint.test.ts.md

```diff
@@ -74,7 +74,33 @@ Generated by [AVA](https://avajs.dev).
               description: '',
               isDeprecated: false,
               name: 'external_type',
-              type: 'string',
+              type: 'enum',
+              values: [
+                {
+                  name: 'pti_site',
+                },
+                {
+                  name: 'alta_org',
+                },
+                {
+                  name: 'salto_site',
+                },
+                {
+                  name: 'brivo_account',
+                },
+                {
+                  name: 'hid_credential_manager_organization',
+                },
+                {
+                  name: 'visionline_system',
+                },
+                {
+                  name: 'assa_abloy_credential_service',
+                },
+                {
+                  name: 'latch_building',
+                },
+              ],
```

Comment:
Are those supposed to be included? Probably not in that manner?

Reply:
This is correct!